ts/IterableInboxEmptyState.tsx

```diff
@@ -1,10 +1,10 @@
 "use strict";
-import React, { Component } from "react";
-import { View, Text, StyleSheet } from "react-native";
+import React, { Component } from "react"
+import { View, Text, StyleSheet } from "react-native"
 
 const IterableInboxEmptyState = () => {
-   const title = "No saved messages";
-   const subtitle = "Check again later!";
+   const title = "No saved messages"
```

Comment:
semicolons are your friend 😆

Reply:
Tapash told me to leave semicolons out since it is typescript